bitnami/kafka/templates/broker/hpa.yaml

```diff
@@ -3,12 +3,11 @@ Copyright Broadcom, Inc. All Rights Reserved.
 SPDX-License-Identifier: APACHE-2.0
 */}}
 
-{{- $replicaCount := int .Values.broker.replicaCount }}
-{{- if and (gt $replicaCount 0) .Values.broker.autoscaling.hpa.enabled }}
+{{- if .Values.broker.autoscaling.hpa.enabled }}
 apiVersion: {{ include "common.capabilities.hpa.apiVersion" ( dict "context" $ ) }}
 kind: HorizontalPodAutoscaler
 metadata:
-  name: {{ printf "%s-broker" (include "common.names.fullname" .) }}
+  name: {{ template "kafka.broker.fullname" . }}
   namespace: {{ include "common.names.namespace" . | quote }}
   labels: {{- include "common.labels.standard" ( dict "customLabels" .Values.commonLabels "context" $ ) | nindent 4 }}
     app.kubernetes.io/component: broker
```

Comment:
broker.autoscaling.hpa.annotations is not  in values.yaml

Reply:
I see similar issues with controller.autoscaling, please check

Reply:
done at https://github.com/bitnami/charts/pull/32516/commits/b644b040e2ce56a4e6fc78f2ff1138e34334e200